src/org/sosy_lab/cpachecker/cfa/types/MachineModel.java

```diff
@@ -1053,10 +1122,8 @@ public BigInteger calculateNecessaryBitfieldOffset(
   @Deprecated
   public BigInteger calculatePaddedBitsize(
       BigInteger pBitOffset,
-      BigInteger pSizeOfConsecutiveBitFields,
```

Comment:
Please recheck whether this parameter is really unnecessary. Is it correctly process struct with several consecutive bitfields?

Reply:
As far as i've tested, it works fine. I had to add quite a bit of logic to the method, so i flatten it.

The reason for removing `sizeOfConsecutiveBitFields` is the following.
The variable stored consecutive bit-fields with padding on the edges, but all of this could be added to `bitOffset` in the first place. In a few methods `pSizeOfConsecutiveBitFields` is just added to the `bitOffset`, which hints the same.